src/Dlp/V2beta1/Gapic/DlpServiceGapicClient.php

```diff
@@ -183,13 +166,54 @@ private static function getLongRunningDescriptors()
 
     private static function getGapicVersion()
     {
-        if (file_exists(__DIR__.'/../VERSION')) {
-            return trim(file_get_contents(__DIR__.'/../VERSION'));
-        } elseif (class_exists('\Google\Cloud\ServiceBuilder')) {
-            return \Google\Cloud\ServiceBuilder::VERSION;
-        } else {
-            return;
+        if (!self::$gapicVersionLoaded) {
+            if (file_exists(__DIR__.'/../VERSION')) {
+                self::$gapicVersion = trim(file_get_contents(__DIR__.'/../VERSION'));
+            } elseif (class_exists('\Google\Cloud\Version')) {
+                self::$gapicVersion = \Google\Cloud\Version::VERSION;
```

Comment:
We should do this wherever possible, but I don't think we can here because `Version` may not exist, and for `RetrySettings` and `OperationResponse` I believe they are not used anywhere outside comments.

Reply:
Importing a namespace that conditionally exists does not cause a problem.

Reply:
Why does the fact they are only used in documentation matter?

Reply:
So this may not be completely right, but I think that `use` statements that are not used in actual code get stripped out by formatting tools that we apply like `phpcbf`. So then we would end up not being able to resolve the types.

This is likely just a practical problem though, and we should figure out how to adjust our formatting tools so that they don't cause this problem, and confirm that docs generate correctly without FQ names.

I won't try to do this now, but we can create an issue for future updates.